cmd/crio/main.go

```diff
@@ -327,9 +327,9 @@ func main() {
 			Value:  lib.DefaultHooksDirPath,
 			Hidden: true,
 		},
-		cli.StringSliceFlag{
-			Name:   "default-mounts",
-			Usage:  "add one or more default mount paths in the form host:container",
+		cli.StringFlag{
+			Name:   "default-mounts-file",
+			Usage:  "path to default mounts file",
```

Comment:
This is a breaking api change, you should actually add a new option and we need to support the old one at least for 2 to 3 releases.

Reply:
Reverted to have the deprecated flag still there.